site/ee_EE.all.json

```diff
@@ -1,602 +1,602 @@
+  "classification_outro": { "other": "" },
+
+  "cookie_banner_message": { "other": "Kasutuskogemuse parandamiseks kasutab see leht küpsiseid. Lehte edasi kasutades nõustud <a href=\"{{.CookiePolicyURL}}\">küpsiste kasutamisega</a>." },
+  "cookie_banner_accept": { "other": "Nõustu" },
+
+  "all_rights_reserved": { "other": "Kõik õigused kaitstud. Ühtegi selle saidi osa ei tohi reprodutseerida ilma meie kirjaliku loata." },
+
+  "users_gender_none": { "other": "Määramata" },
+  "users_gender_female": { "other": "Naine" },
+  "users_gender_male": { "other": "Mees" },
+  "users_gender_diverse": { "other": "Sooneutraalne" },
+  "users_gender_other": { "other": "Muu" },
+
+
+  "pricing_quality_hd": { "other": "HD" },
+  "pricing_quality_sd": { "other": "SD" },
+  "pricing_quality_hd_only": { "other": "Ainult HD" },
+  "pricing_quality_sd_only": { "other": "Ainult SD" },
+
+  "shopping_card_save_card": { "other": "Hoidke see kaart edaspidiseks kasutamiseks alles" },
+  "shopping_card_button_change": { "other": "Kuva kõik salvestatud kaardid" },
+  "shopping_card_existing_description": { "other": "{{.Card.Brand}}, mis lõpeb numbritega {{.Card.Number}}" },
+  "shopping_card_existing_expiry": { "other": "(Aegub {{.Card.Expiry}})" },
+  "shopping_card_existing_expired": { "other": "(aegunud)" },
+  "shopping_card_use_other": { "other": "Kasutage uut kaarti" },
+  "shopping_card_use_new_card": { "other": "Vaheta kaarti" },
+
+  "shopping_info_ownership_plan": { "other": "Tellimus" },
+
+  "shopping_action_credit": { "other": "Täida" },
+
+  "shopping_info_rental_period_coming_soon": { "other": "alates avaldamise kuupäevast ja kellaajast." },
+
+
+  "usersubscriptions_change_payment_method_change": { "other": "Muuda krediitkaarti" },
+  "usersubscriptions_change_payment_method_modal_title": { "other": "Värskendage krediitkaardi andmeid" },
+  "usersubscriptions_change_payment_method_modal_submit": { "other": "Värskenda" },
+
+  "usersubscriptions_subscription_expiry": { "other": "Aegub {{.Expiry}}" },
+  "usersubscriptions_subscription_status_cancelled": { "other": "usersubscriptions_subscription_status_cancelled" },
+  "usersubscriptions_subscription_status_errored": { "other": "usersubscriptions_subscription_status_errored" },
+  "usersubscriptions_subscription_status_past_due": { "other": "usersubscriptions_subscription_status_past_due" },
+  "usersubscriptions_subscription_status_trialing": { "other": "Prooviperiood" },
+  "usersubscriptions_subscription_unsubscribe": { "other": "Tühista minu tellimus" },
+  "usersubscriptions_unsubscribe_modal_cancel": { "other": "Tühista" },
+  "usersubscriptions_unsubscribe_modal_confirm": { "other": "Jah, tühistage minu tellimus" },
+  "usersubscriptions_unsubscribe_modal_title": { "other": "Kas tühistada tellimus?" },
+  "usersubscriptions_unsubscribe_modal_body": { "other": "See tühistab paketi {{.Name}} tellimuse. Saate vaadata paketi sisu, kuni see aegub {{.Aeguv}}." },
```

Comment:
i think you might have translated the variable here

Reply:
{{.Aeguv}}

Reply:
Damn I thought I caught all of those.